internal/api/download_test.go

```diff
@@ -0,0 +1,119 @@
+package api
+
+import (
+	"bytes"
+	"encoding/json"
+	"github.com/opg-sirius-finance-admin/internal/model"
```

Comment:
If you haven't already, get https://pkg.go.dev/golang.org/x/tools/cmd/goimports for formatting. Otherwise maybe it needs updating?